app/code/Magento/Bundle/view/base/web/js/price-bundle.js

```diff
@@ -375,6 +375,17 @@ define([
         var tiers = optionConfig.tierPrice,
             magicKey = _.keys(oneItemPrice)[0],
             lowest = false;
+        
+        //tiers is undefined when options has only one option 
+		if (undefined == tiers) {
+		    var firstKey = Object.keys(optionConfig)[0];
```

Comment:
According to [Magento Coding Standard](https://devdocs.magento.com/guides/v2.2/coding-standards/code-standard-javascript.html) all variable declaration should be at the start in the function.

Reply:
thanks, have updated accordingly.